operations/use-syslog-addon.yml

```diff
@@ -0,0 +1,25 @@
+- type: replace
+  path: /addons?/-
+  value:
+    name: syslog_forwarder
+    include:
+      stemcell:
+      - os: ubuntu-trusty
+    jobs:
+    - name: syslog_forwarder
+      release: syslog
+    properties:
+      syslog:
+        address: ((syslog_remote_address))
+        port: ((syslog_remote_port))
+        transport: tcp
+        fallback_servers: []
+        custom_rule: ""
```

Comment:
Can we expose `transport`, `fallback_servers`, and `custom_rule` as we do in the runtime-config? We can go with names like the one's you've used (`syslog_transport`, etc.).

Reply:
The new names are superior, but changing the names will require some communication to current users.  We should think about a means of deprecating existing ops files, maybe.